fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Election.java

```diff
@@ -189,8 +208,14 @@ public String computerRegisteredVotes() {
     // Since messageMap is a TreeMap, votes will already be sorted in the alphabetical order of
     // messageIds
     for (PublicKey senderPk : messageMap.values()) {
-      for (ElectionVote vote : voteMap.get(senderPk)) {
-        listOfVoteIds.add(vote.getId());
+      if (getElectionVersion() == ElectionVersion.OPEN_BALLOT) {
+        for (ElectionVote vote : openVoteMap.get(senderPk)) {
+          listOfVoteIds.add(vote.getId());
+        }
+      } else {
+        for (ElectionEncryptedVote vote : encryptedVoteMap.get(senderPk)) {
+          listOfVoteIds.add(vote.getId());
+        }
       }
```

Comment:
```suggestion
     Map<PublicKey, List<ElectionVote>> voteMap = getElectionVersion() == ElectionVersion.OPEN_BALLOT ? openVoteMap :  encryptedVoteMap;
        for (ElectionVote vote :  voteMap.get(senderPk)) {
          listOfVoteIds.add(vote.getId());
        }
```

Reply:
I think it is pretty messy the way it is done right now, here is a suggestion to make it more clean

Reply:
It won't work, because openVoteMap is List<ElectionVote> and encryptedVoteMap is List<ElectionEncryptedVote> but thanks for suggestions !

Reply:
l.44-46 says that `openVoteMap `and` encryptedVoteMap` are Maps, and calling a variable that ends with Map in the naming is bad practice :) 
```
  private final Map<PublicKey, List<ElectionVote>> openVoteMap;
  // Map that associates each sender pk to their encrypted votes
  private final Map<PublicKey, List<ElectionEncryptedVote>> encryptedVoteMap;
```

Reply:
oh I see, i refactored the name !

Reply:
I might not see what you are refering to but at line 44 and 46 I say that the 2 variables `openVoteMap` and `encryptedVoteMap` are declared as maps, so no need to rename IMO since the list is obtainged with `openVoteMap.get(senderPk)` for instance